dlib/dnn/layers.h

```diff
@@ -1587,6 +1722,14 @@ namespace dlib
                 // ignore other layer types
             }
 
+            template <typename U, typename E>
+            void set_input_no_bias(add_layer<layer_norm_, U, E>& l)
+            {
+                disable_bias(l.subnet().layer_details());
+                set_bias_learning_rate_multiplier(l.subnet().layer_details(), 0);
+                set_bias_weight_decay_multiplier(l.subnet().layer_details(), 0);
+            }
+
```

Comment:
I think it makes sense to update this visitor, to remove bias from inputs of `layer_norm`. However, I think it should be renamed. Something like: `remove_superfluous bias`, so that it can still make sense if we add more layers that can be added to the visitor. Any better idea?

Reply:
Yeah, how about `remove_duplicative_biases`?

Reply:
`disable_duplicative_biases`? Since we call `disable_bias` on the layers...

Reply:
Oh yeah, that’s a better name :)